docs/guides/using-org-slugs.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: 'Using organization slugs in URLs'
+description: Learn how to use organization slugs in URLs to manage the active Clerk organization.
+---
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Next.js + Clerk application",
+      link: "/docs/quickstarts/nextjs",
+      icon: "nextjs",
+    },
+    {
+      title: "Enable organizations for your instance",
+      link: "/docs/organizations/overview",
+      icon: "globe",
+    },
+    {
+      title: "Explore the full sample repository on GitHub",
+      link: "https://github.com/clerk/orgs/tree/main/examples/sync-org-with-url",
+      icon: "git",
+    }
+  ]}
+>
+  - Allow users to choose an organization slug
+  - Include an organization slug in your URLs
+  - Render organization-specific content
+</TutorialHero>
+
+## Introduction
+
+In this guide, you will learn how to include an organization slug in your application's URLs, and
+how to render information about the user's active organization at runtime.
+
+## Background
+
+With [Organizations](/docs/organizations/overview), users can create entities that represent their companies within your
+application. Users can be members of multiple organizations, but can only have one
+[active organization](/docs/organizations/overview#active-organization) at any given time. Clerk automatically sets organizations as
+active when a user picks an organization with a pre-built component (e.g. the
+[Organization Switcher](/docs/components/organization/organization-switcher)), and Clerk provides
+APIs (like [<code>auth()</code>](/docs/references/nextjs/auth) and [<code>useAuth()</code>](/docs/references/react/use-auth)), that let you determine the active organization and render the appropriate
+content.
+
+A relatively common practice is to represent organization-scoped spaces within an app by prefixing
+URLs with an organization slug. For example, imagine a B2B application named "Petstore" that has
+two customers, "Acmecorp" and "Widgetco". The application may choose to have urls like:
+
+`https://petstore.example.com/orgs/`<b>`acmecorp`</b>`/dashboard` ⬅ indicates **Acmecorp**'s dashboard
+`https://petstore.example.com/orgs/`<b>`widgetco`</b>`/dashboard` ⬅ indicates **Widgetco**'s dashboard
+
+It is also possible to use an [organization ID](/docs/references/javascript/organization/organization#properties) to
+indicate the active organization in URLs:
+
+`https://petstore.example.com/orgs/`<b>`org_1a2b3c4d5e6f7g8e`</b>`/dashboard` ⬅ **Acmecorp's ID** indicates Acmecorp's dashboard
```

Comment:
what do you mean by Acmecorp's ID here? Did you mean Acemecorp's organization ID?

```suggestion
`https://petstore.example.com/orgs/`<b>`org_1a2b3c4d5e6f7g8e`</b>`/dashboard` **Acmecorp's ID** indicates Acmecorp's dashboard
```

Reply:
Oh yes, I mean organization id!

Clarified: f7bb4b5a4e7636f88c702598ba7c39a0ce527249